### ISSUE TITLE
Atualização no. 1 para UI

### ISSUE THREAD

**Issue by WodsonKun** [ROM-PacMe/UXMB#1]
(Hablando en Portugués de Brasil, traduce para Español)
Oi! Sou eu de novo! Agora, vim ajudar na UI do UXMB...

[UXMB (Preview UI Update).zip](https://github.com/ROM-PacMe/UXMB/files/1016313/UXMB.Preview.UI.Update.zip)

O que eu trouxe desta vez:
 • Ícones da XMB do PS3 (Não encontrei do PS4, ainda procurando)
 • Fontes do PS4
 • ICON0 ([boot.png](https://github.com/ROM-PacMe/UXMB/blob/master/img/boot.png) redimensionado):
![icon0](https://cloud.githubusercontent.com/assets/22161127/26277512/92c8dd82-3d5f-11e7-878a-f2e7177579a5.png)

 • PIC1 novo ([bootup.png](https://github.com/ROM-PacMe/UXMB/blob/master/img/bootup.png) redimensionado)
![pic1](https://cloud.githubusercontent.com/assets/22161127/26277519/c4994784-3d5f-11e7-80e2-1a5c3f2ff82c.png)

Logo trarei mais coisas... Aguarde!

**Comment by WodsonKun** [comment]
Ei @ROM-PacMe, onde conseguiu esse .png das waves do PS4? Talvez, se tivesse um .gif... Isso ajudaria muito.

**Comment by bm0x** [comment]
hey! @DarkLegend481 de hecho en el UXMB tengo un gif de un wave de PS4 original
https://github.com/UXMB-Mod/images/blob/master/bootup_ps4.gif?raw=true

**Comment by bm0x** [comment]
PD: creo que está en portugués la presentación

**Comment by bm0x** [comment]
si quieres ayudarme en UXMB, no dudes en aceptar mi invitación a:
[UXMB-Mod](https://github.com/UXMB-Mod)

**Comment by WodsonKun** [comment]
@ROM-PacMe Obrigado pelo convite! Vou ajudar o máximo o possível... Estou pensando em fazer alguns cursos de programação em Lua, para poder ajudar com códigos também

**Comment by WodsonKun** [comment]
(Testando no emulador PPSSPP) As atualizações nos arquivos ICON0 e PIC1 é porque eles ficam desse jeito:
![wp_ss_20170520_0002](https://cloud.githubusercontent.com/assets/22161127/26280433/33e74736-3daa-11e7-9053-765b00388e38.png)

Em minha opinião, precisava ou de novos ICON0 e PIC1 ou de uma bela redimensionada...

**Comment by WodsonKun** [comment]
Ah! Sei de algo que pode ajudar!

Olhe isto, sei que é para o Kodi/XMBC, mas... Vai que nós consigamos algo com isso:
https://m.youtube.com/watch?v=3hk0z1Pjy7s

**Comment by bm0x** [comment]
de hecho el único código Lua que se usa es del interprete de ONELua :3

**Comment by bm0x** [comment]
[ONELua Documentation - English](http://onelua.x10.mx/psp/docs/en/index.html)

**Comment by WodsonKun** [comment]
[GIF da UI do PS4](https://github.com/UXMB-Mod/images/blob/master/bootup_ps4.gif)

Isso é um gif da UI do PS4, com a wave no fundo... 

Precisamos de uma wave assim:
https://github.com/ROM-PacMe/UXMB/blob/master/img/bootup.png

Sem UI, só a wave mesmo...

**Comment by WodsonKun** [comment]
@ROM-PacMe Acontece que eu não sei NADA sobre programação para o PSP... :\ Daí fica difícil...

**Comment by bm0x** [comment]
ONELua es el interprete basado en el código Lua para el código C de los sistemas de Play Station

**Comment by bm0x** [comment]
todo mi código es basado en el mismo interprete

**Comment by bm0x** [comment]
`screen.print(100,200,"hello World")
while true do
buttons.read()
if buttons.cross then
dofile("hello.lua")
end
end`

**Comment by WodsonKun** [comment]
Eu dei uma lida... Não é tão difícil, eu tinha pensado que era um bicho de sete cabeças kkkkkkk

**Comment by bm0x** [comment]
xDDDD

**Comment by WodsonKun** [comment]
@ROM-PacMe Agora me veio uma questão na cabeça... Vai ser apenas o XMB do PS4 ou TODA a interface do PS4, incluindo os "quadrados", as pastas, etc...?

**Comment by bm0x** [comment]
quiero intentar hacer todo lo posible para hacerlo muy semejante a PS4...

**Comment by WodsonKun** [comment]
Bom... Como vamos fazer para corrigir o bootloop e iniciar a UI?

**Comment by WodsonKun** [comment]
Achei uma wave, é do PS3, mas por enquanto deve servir...

https://m.youtube.com/watch?v=5gAsmP1zrOE

**Comment by bm0x** [comment]
eso ya está hecho jeje ;)

**Comment by bm0x** [comment]
![18425251_1933732603541761_5479599363485010187_n](https://cloud.githubusercontent.com/assets/16091640/26280685/78bfa1a8-3dac-11e7-8d07-f69027a84eed.jpg)
![18422782_1933733506875004_6014443023271584229_o](https://cloud.githubusercontent.com/assets/16091640/26280686/78ed0238-3dac-11e7-9d84-bf7fcd5c4b34.jpg)
 :3

**Comment by bm0x** [comment]
![screenshot 2017-05-20 at 22 30 16](https://cloud.githubusercontent.com/assets/16091640/26280689/83269f66-3dac-11e7-8269-faf8aff7f5ff.png)
n.n

**Comment by WodsonKun** [comment]
Bom, acho que amanhã eu começo a ajudar com o código... Vou fazer isso pelo meu Lumia, já que o único PC aqui em casa é da minha mãe 😂😒...

**Comment by WodsonKun** [comment]
Minha bateria está acabando... Te encontro aqui amanhã?

**Comment by bm0x** [comment]
claro! n.n
pronto lanzaré un nuevo release ya booteable para que lo pruebes :3 
🥇

**Comment by WodsonKun** [comment]
OK... Acho que vou tentar um sistema de detecção de UMDs, para inicia-los direto do UXMB... Talvez, até um sistema de detecção de GameID (SLUS, SLES, etc...) para uso de imagens...

Boa noite! 😉

**Comment by bm0x** [comment]
nuevas noches n.n
y no te preocupes, que para el sistema de archivos y UMD me las arreglo con el código lua genérico :3

**Comment by WodsonKun** [comment]
Bom dia! Testei a nova release do UXMB e ficou ótima cara! Mas eu acho que nós deviamos remover aquela última tela 
[Essa tela aqui](https://cloud.githubusercontent.com/assets/16091640/26280686/78ed0238-3dac-11e7-9d84-bf7fcd5c4b34.jpg)

Acho que a UI deveria bootear direto na wave, igual no PS4...

**Comment by WodsonKun** [comment]
Vou fazer upload dos ícones do XMB no repositório de imagens do UXMB... Tudo bem se eu fizer isso?

**Comment by WodsonKun** [comment]
Eu tô tendo alguns probleminhas... Toda vez que eu edito os arquivos, o ONEdebugger aparece com um erro...

"Unexpected ' near !"

É sempre isso que aparece...

**Comment by bm0x** [comment]
en primer lugar... has intentado ver como la la lógica del ONELua? hay partes en donde no se puede tan fácilmente...

**Comment by bm0x** [comment]
partes en donde... el código interpretado tiene más lógica que código colocado de por si en el sistema...

**Comment by bm0x** [comment]
y lo del inicio... puedo arreglarlo fácilmente... así que en eso no hay ningún problema..

**Comment by bm0x** [comment]
lo que haré será utilizar el wave splash en el modo init, para poder obtener algo parecido

**Comment by bm0x** [comment]
![image](https://cloud.githubusercontent.com/assets/16091640/26285530/ee73b65a-3e1f-11e7-88fd-c38d582578fa.png)

**Comment by bm0x** [comment]
![image](https://cloud.githubusercontent.com/assets/16091640/26285832/853dd89a-3e25-11e7-8d7b-70ac9165ed40.png)

**Comment by WodsonKun** [comment]
Sim, eu li e ví um pouco da lógica do ONELua, mas mesmo assim... Fiz apenas algumas edições no texto da Recovery do UXMB e tive esse erro.

Já mando uma screenshot, espera aí...

**Comment by bm0x** [comment]
ok

**Comment by WodsonKun** [comment]
![wp_ss_20170521_0002](https://cloud.githubusercontent.com/assets/22161127/26286566/3a5e2676-3e3f-11e7-8bcc-d3b1b04db9d1.png)
![wp_ss_20170521_0003](https://cloud.githubusercontent.com/assets/22161127/26286567/3a61df8c-3e3f-11e7-9866-eaf626e357dd.png)

**Comment by bm0x** [comment]
muestrame que ediciones has hecho

**Comment by WodsonKun** [comment]
Editei o código com o [Serris Code Editor BETA](https://www.microsoft.com/store/apps/9nblggh4rm22)

**Comment by WodsonKun** [comment]
![wp_ss_20170521_0004](https://cloud.githubusercontent.com/assets/22161127/26286627/4ce50d4a-3e40-11e7-942c-c1050a938828.png)
![wp_ss_20170521_0005](https://cloud.githubusercontent.com/assets/22161127/26286628/4ce662f8-3e40-11e7-88c9-51ec1cb26e28.png)
![wp_ss_20170521_0006](https://cloud.githubusercontent.com/assets/22161127/26286629/4ce6db5c-3e40-11e7-9d9c-071ecfa65b17.png)

Fiz só coisinhas básicas, nada envolvendo muito código...

**Comment by bm0x** [comment]
que raro...

**Comment by WodsonKun** [comment]
Só pra ver como funcionava o ONELua e se a minha edição iria funcionar...

**Comment by WodsonKun** [comment]
Vou tentar re-escrever o código aqui... Quando terminar, eu aviso

**Comment by bm0x** [comment]
https://www.youtube.com/watch?v=ficzkQ9uZtI&feature=youtu.be

**Comment by WodsonKun** [comment]
Ah, você tem como compilar o projeto... Eu não, faço tudo pelo meu Lumia, já que minha mãe não empresta o computador dela 😒

Acho que eu vou ter que MANDAR o código em um .zip e VOCÊ vai ter que compilar... 🙄🌬

**Comment by bm0x** [comment]
no, la compilación solo consiste en cambiar la extensión .lua a .pacme y pasarlo a un directorio compilado, no es nada más jeje

**Comment by WodsonKun** [comment]
Ah, ta... E se eu adicionar arquivos e trocar o diretório de alguns... Eles ainda funcionam?

**Comment by bm0x** [comment]
si, pero tienes que hacer que concuerden directorios y demás
![image](https://cloud.githubusercontent.com/assets/16091640/26287029/66d4319a-3e40-11e7-9317-8ed16be38d48.png)

**Comment by WodsonKun** [comment]
AAAAHHHHHHHHH! Fiz um simples edição DENOVO, e ele dá o mesmo erro, agora no script.lua

**Comment by bm0x** [comment]
Oye, me pasas PPSSP para Windows Mobile? No lo encuentro por ninguna parte jeje...

**Comment by WodsonKun** [comment]
[PPSSPP-GO-IGOR SANCHES.zip](https://github.com/ROM-PacMe/UXMB/files/1017343/PPSSPP-GO-IGOR.SANCHES.zip)

Precisa do Interop Tools para instalar
Obs.: Nesta versão, ele não usa JIT

**Comment by WodsonKun** [comment]
Olha, talvez eu fique algum tempo sem aparecer por aqui... (Brigas aqui em casa...)
Mas sempre que der, vou acompanhar o projeto 😉

**Comment by WodsonKun** [comment]
@ROM-PacMe Conseguiu instalar o PPSSPP?

**Comment by bm0x** [comment]
si... pero no pude correr el juego, se me salía el emulador...

**Comment by WodsonKun** [comment]
No meu Lumia 635 (512MB de RAM, mas aumentei para 1GB kkkkk) o God of War, por exemplo, de 60FPS roda a 2FPS... 😐

**Comment by WodsonKun** [comment]
Mas você desligou o JIT?

**Comment by WodsonKun** [comment]
@ROM-PacMe Ah, para rodar no meu, tive que atualizar para a Creators Update usando a RM do 950XL...

E sobre o UXMB... Terminei de re-escrever ele, mas ainda sim ele não rodou
